src/js/components/Anchor/StyledAnchor.js

```diff
@@ -13,8 +13,11 @@ const StyledAnchor = styled.a`
   font-size: inherit;
   line-height: inherit;
   color: ${props =>
-    (props.theme.dark ? props.theme.global.colors.darkBackground.text
-      : props.theme.anchor.color)};
+    (props.theme.dark ? (
+      props.theme.anchor.colors.dark || props.theme.global.colors.darkBackground.text
+    ) : (
+      props.theme.anchor.colors.light || props.theme.global.colors.darkBackground.text
```

Comment:
`darkBackground` doesn't look correct here.

Reply:
yup yup

let me fix it